R/ptype-abbr-full.R

```diff
@@ -32,15 +32,25 @@
 #' cat(vec_ptype_abbr(1:10))
 vec_ptype_full <- function(x, ...) {
   check_dots_empty0(...)
-  UseMethod("vec_ptype_full")
```

Comment:
I swear we needed to keep `UseMethod()` in there at the end of the function because of something to do with R CMD check, even though it can't be called. Like how `vec_ptype()` is set up:

```r
> vec_ptype
function (x, ..., x_arg = "", call = caller_env()) 
{
    check_dots_empty0(...)
    return(.Call(ffi_ptype, x, x_arg, environment()))
    UseMethod("vec_ptype")
}
```

Reply:
I think it's so that roxygen2 knows it's a generic.